apis/v1/grpcroute_types.go

```diff
@@ -0,0 +1,618 @@
+/*
```

Comment:
For the record, I used `git mv` here to try to maintain git history. That appears to have worked everywhere but the GitHub UI.
<img width="1004" alt="Screenshot 2024-04-01 at 3 18 01 PM" src="https://github.com/kubernetes-sigs/gateway-api/assets/419322/21cf0797-7e6c-4efb-a30e-7683b37b0e22">